### PR TITLE
School Finder State/Province label

### PR DESCRIPTION
**Pull request by aaronschachter**
Updates help text per #3321.  Will display as "State" in US, but "Province" in Canada.

### CODE REVIEW

File: lib/modules/dosomething/dosomething_signup/dosomething_signup.signup_data_form.inc [hunk 1]
```diff
@@ -374,12 +374,19 @@ function dosomething_signup_user_signup_data_form($form, &$form_state, $signup)
     $form['school_finder']['help_link'] = array(
       // Used for front-end autocomplete.
       '#type' => 'markup',
```
Comment (sergiitk) [inline lib/modules/dosomething/dosomething_signup/dosomething_signup.signup_data_form.inc:376]: I know it's not documented, but there's should be special form type called [link](https://api.drupal.org/comment/16104#comment-16104). Can you check if this is still working?

Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_signup/dosomething_signup.signup_data_form.inc:376]: I've tried it.. but when I click on "Need help locating your school?"  the actual link `href` is loaded, e.g. I am redirected to http://dev.dosomething.org:8888/%23.

Comment (sergiitk) [inline lib/modules/dosomething/dosomething_signup/dosomething_signup.signup_data_form.inc:376]: Ok then, it's minor anyways.

Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_signup/dosomething_signup.signup_data_form.inc:376]: Right on, was worth a shot.  Ok to ship?

**Comment by sergiitk** [comment]
:+1:

**Pull request merged**